plenum/common/ledger_manager.py

```diff
@@ -483,6 +489,8 @@ def _make_consistency_proof(self, ledger, end, catchup_till):
     @measure_time(MetricsName.PROCESS_CATCHUP_REP_TIME)
     def processCatchupRep(self, rep: CatchupRep, frm: str):
         logger.info("{} received catchup reply from {}: {}".format(self, frm, rep))
+        if frm in self.wait_catchup_rep_from:
```

Comment:
Maybe will be better use ```recvdCatchupRepliesFrm```. But in this case we will not check nodes which stopped to answer during catchup.